packages/web3-core/src/AbstractWeb3Module.js

```diff
@@ -76,7 +76,7 @@ export default class AbstractWeb3Module {
     }
 
     /**
-     * Setter for the defaultAccount property
+     * Setter for the defaultBlock property
```

Comment:
This was a typo.

Reply:
It's a setter for defaultBlock and defaultAccount.

Reply:
I just left this for anyone else reviewing this and wondering: Why does this need to be changed. The answer is: You corrected a typo (leftover from copy-paste I assume).